xc/device.ml

```diff
@@ -1408,6 +1416,8 @@ module Vusb = struct
     if device_model = Profile.Qemu_trad then
       raise (Internal_error (Printf.sprintf "Failed to plug VUSB %s because domain %d uses device-model profile %s." id domid (Profile.string_of device_model)));
     debug "Vusb plugged: vusb device %s plugged" id;
+    (* Need to reset USB device before passthrough to vm according to CP-24616 *)
+    call_usb_reset [ hostbus ^ "-" ^ hostport ];
```

Comment:
I find these strings that specify the usb device to be a bit opaque. I'd prefer it if we kept the bus/port arguments right up until we had to turn them into this string - preferably the script would take the bus/port as arguments, but, if we can't do that, at least our `call_usb_reset` function should.

Reply:
yes, now the script just takes path as a parameter.